Shared/Samples/Find route around barriers/FindRouteAroundBarriersView.Views.swift

```diff
@@ -16,43 +16,47 @@ import ArcGIS
 import SwiftUI
 
 extension FindRouteAroundBarriersView {
-    /// The list of settings for the sample.
-    struct SettingsList: View {
-        /// The view model for the sample.
-        @ObservedObject var model: Model
+    /// A list of settings for modifying route parameters.
+    struct RouteParametersSettingsList: View {
```

Comment:
I think this should be just `RouteParametersSettings`
```suggestion
    struct RouteParametersSettings: View {
```